lib/components/user/saved-trip-screen.js

```diff
@@ -180,10 +179,15 @@ class SavedTripScreen extends Component {
           validateOnChange={false}
           validateOnBlur
           validationSchema={validationSchema}
-          onSubmit={isCreating ? this._handleSaveNewTrip : this._handleSaveTripEdits}
+          onSubmit={this._handleTrip}
```

Comment:
I think this is supposed to be `_handleSaveTrip` (!).

Reply:
Wow good catch!